tools/install_bazel.sh

```diff
@@ -21,6 +20,6 @@ else
     OS=$(uname -s | tr '[:upper:]' '[:lower:]')
     ARCH=$(uname -m) && [ "$ARCH" != "aarch64" ] || ARCH="arm64"
     echo "Downloading bazel-${BAZEL_VERSION}-${OS}-${ARCH} to ${BAZEL}"
```

Comment:
This should be updated to match the actual download.

Reply:
I have updated the script to lock down the bazelisk version, bazel version will be just auto detected by bazelisk.